jwst/tests_nightly/general/pipelines/test_nrc_image3_1.py

```diff
@@ -0,0 +1,103 @@
+import os
+import pytest
+from astropy.io import fits as pf
+from jwst.pipeline.calwebb_image3 import Image3Pipeline
+
+pytestmark = [
+    pytest.mark.usefixtures('_jail'),
+    pytest.mark.skipif(not pytest.config.getoption('bigdata'),
+                       reason='requires --bigdata')
+]
+
+
+def test_image3_pipeline1(_bigdata):
+    """Regression test definitions for CALIMAGE3 pipeline.
+
+    Regression test of calwebb_image3 pipeline on NIRCam
+    simulated long-wave data.
+
+    """
+    subdir = os.path.join(_bigdata, 'pipelines', 'nircam_calimage3')
+    ignore_kws = ['DATE', 'CAL_VER', 'CAL_VCS', 'CRDS_VER', 'CRDS_CTX']
+
+    try:
+        os.remove("nrca5_47Tuc_subpix_dither1_newpos_a3001_crf.fits")
```

Comment:
I am guessing these are output files?

Reply:
Yes, many of the tests remove old versions of their output files before running. That way you know you're starting cleanly. There are occasions where a change in behavior causes a step or test to put its output in a file of a different name, so if the old one is still there, it gets used when doing the comparison and you don't know that something's changed.

Reply:
OK, but as stated above, this is no longer necessary with `pytest + tmpdir` usage.